src/wasm-linker.cpp

```diff
@@ -381,12 +381,14 @@ void Linker::makeDynCallThunks() {
   }
 }
 
-Function* Linker::generateImportThunk(Name name, const FunctionType* funcType) {
+Function* Linker::getImportThunk(Name name, const FunctionType* funcType) {
+  Name thunkName = std::string("__importThunk_") + name.c_str();
```

Comment:
I guess technically having duplicate functions with the same name is not a correctness issue if they have different indexes, although it's a problem for wast files, and it causes errors in sexpr-wasm and wasm-as.